### ISSUE TITLE
Custom date/time format

### ISSUE THREAD

**Issue by mpalczew** [pawelmalak/flame#277]
I think there should be a way to do a custom date / time format. 

e.g. my use case. 
date should have comma after day.  e.g. January 3, 2022,  but also I prefer short month.  So Jan 3, 2022. 

As far as time. It uses 24 hour time instead of am/pm with no way of changing that.

**Comment by pawelmalak** [comment]
You can set custom months in the settings.

**Comment by EVIL-Teken** [comment]
Would like to add my voice to wanting to see both 12 vs 24 hour time supported via check box.

**Comment by ldlar** [comment]
I agree on wanting to see both time formats supported by check box.  Thanks for a great app.
